README.md

```diff
@@ -14,125 +19,147 @@ predictions, and insights form noisy, incomplete, and erroneous data.
 \c holo
-ALTER SCHEMA public OWNER TO holo;
+ALTER SCHEMA public OWNER TO holocleanuser;
 ```
 
-In general, to connect to the `holo` database from the Postgres psql console
-```
-\c holo
-```
+You can connect to the `holo` database from the PostgreSQL `psql` console by running
+`psql -U holocleanuser -W holo`.
 
 HoloClean currently populates the database `holo` with auxiliary and meta tables.
-To clear the database simply connect as a root user or as `holocleanuser` and run
-```
+To clear the database simply connect as a `root` user or as `holocleanuser` and run
+```sql
 DROP DATABASE holo;
 CREATE DATABASE holo;
 ```
 
-### 3. Set up HoloClean
+#### Option 2: Using a Docker Container
+If you are familiar with docker, an easy way to start using
+HoloClean is to start a PostgreSQL docker container.
+
+To start a PostgreSQL docker container, run the following command:
+
+```bash
+docker run --name pghc \
+    -e POSTGRES_DB=holo -e POSTGRES_USER=holocleanuser -e POSTGRES_PASSWORD=abcd1234 \
+    -p 5432:5432 \
+    -d postgres:11
+```
+
+which starts a backend server and creates a database with the required permissions.
+
+You can then use `docker start pghc` and `docker stop pghc` to start/stop the container.
 
-#### Virtual Environment
 
-##### Option 1: Set up a conda Virtual Environment
+Note the port number which may conflict with existing PostgreSQL servers.
+Read more about this docker image [here](https://hub.docker.com/_/postgres/). 
 
-Install Conda using one of the following methods
+### 2. Setting up HoloClean
+HoloClean runs on Python 2.7 or 3.6+. We recommend running it from within
+a virtual environment.
 
-##### Ubuntu
+#### Creating a virtual environment for HoloClean
+##### Option 1: Conda Virtual Environment
 
-For **32-bit machines** run
+First, install Conda using one of the following methods:
+
+* Ubuntu *32-bit machines* run
 ```bash
 $ wget https://repo.continuum.io/archive/Anaconda-2.3.0-Linux-x86.sh
 $ sh Anaconda-2.3.0-Linux-x86.sh
 ```
-
-For **64-bit machines** run
+* Ubuntu *64-bit machines* run
 ```bash
 $ wget https://repo.continuum.io/archive/Anaconda-2.3.0-Linux-x86_64.sh
 $ sh Anaconda-2.3.0-Linux-x86_64.sh
 ```
 
-##### MacOS
-
-Follow instructions [here](https://conda.io/docs/user-guide/install/macos.html) to install
-Anaconda (NOT miniconda).
-
-##### Create a conda environment
+* macOS, please follow the instructions [here](https://conda.io/projects/conda/en/latest/user-guide/install/macos.html) to install
+Anaconda (not miniconda).
 
-Create a **Python 2.7** conda environment by running
+Second, create a conda environment (python 2.7 or 3.6+).
+For example, to create a *Python 3.6* conda environment, run:
 
 ```bash
-$ conda create -n holo_env python=2.7
+$ conda create -n hc36 python=3.6
 ```
 
 Upon starting/restarting your terminal session, you will need to activate your
 conda environment by running
 ```bash
-$ source activate holo_env
+$ source activate hc36
```

Comment:
Consider adding instructions to deactivate venv when finished with `source deactivate`